test/zuliprc

```diff
@@ -1,4 +1,4 @@
 [api]
-email=cordelia@zulip.com
-key=wlueAg7cQXqKpUgIaPP3dmF4vibZXal7
-site=http://localhost:9991
+email=api-test-bot@chat.zulip.org
+key=K1w7xuUlCorRXjDCz2EkCC4IXMbdYOpK
+site=https://chat.zulip.org
```

Comment:
Tests were already failing for several builds, so my commit didn't make any difference to that. First commit is manually tested and merge-able.